### PR TITLE
 Add FacilityAlias model 

### PR DESCRIPTION
**Pull request by jwalgran**
## Overview

This tables allows us to keep track of the OAR IDs for facilities that are merged with other facilities or deleted so that we can report on their removal and redirect to their replacement facilities.

We applied the same primary key field settings to `oar_id` that were used for `Facility.id` because we only want a single alias for each OAR ID and we do not want it to change.

Connects #605

## Testing Instructions

* Verify that running `./scripts/update` applies the migration and creates the new table.

## Checklist

- [x] `fixup!` commits have been squashed
- [ ] CI passes after rebase
- [X] ~CHANGELOG.md updated with summary of features or fixes, following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guidelines~ Skipped because this is an internal change.

### CODE REVIEW

File: src/django/api/models.py [hunk 1]
```diff
@@ -740,3 +740,37 @@ class Meta:
     def __str__(self):
         return '{0} - {1} - {2}'.format(self.facility_list_item, self.facility,
                                         self.status)
+
+
+class FaclilityAlias(models.Model):
```
Comment (kellyi) [inline src/django/api/models.py]: I set this up and it worked except that when I ran this in `shell_plus` it reported an error.

```
>>> FacilityAlias.objects.all()
Traceback (most recent call last):
  File "<console>", line 1, in <module>
NameError: name 'FacilityAlias' is not defined
```

Turns out there's a typo in the name of this model: `FaclilityAlias` -> `FacilityAlias`

Comment (jwalgran) [reply src/django/api/models.py]: Sorry for not giving this a more thorough test. Will fix.

Comment (jwalgran) [reply src/django/api/models.py]: I fixed the typo in 974d8da.

**Comment by jwalgran** [comment]
This is ready for another look. I added a `__str__` method to make shell testing more clear.

File: src/django/api/models.py (hunk 1 above)
Comment (kellyi) [reply src/django/api/models.py]: No worries! Checked out the newer commits and everything's working as expected now.

**Comment by jwalgran** [comment]
Thanks for catching my goof.

**Pull request merged**